ai_worker/src/parsers/text.py

```
@@ -0,0 +1,108 @@
+        """
+        self._validate_file_exists(filepath)
+
+        extension = self._get_file_extension(filepath)
+        path = Path(filepath)
+
+        if extension == ".txt":
+            content = self._parse_text_file(filepath)
+        elif extension == ".pdf":
+            content = self._parse_pdf_file(filepath)
+        else:
+            raise ValueError(f"Unsupported file format: {extension}")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)